CircleciScripts/check_sdk_on_maven.py

```diff
@@ -47,7 +44,11 @@
     'aws-android-sdk-kinesis', 
     'aws-android-sdk-pinpoint', 
     'aws-android-sdk-machinelearning', 
-    'aws-android-sdk-sqs'
+    'aws-android-sdk-sqs',
+    'aws-android-sdk-connect',
+    'aws-android-sdk-connectparticipant',
+    'aws-android-sdk-textract',
+    'aws-android-sdk-kinesisvideo-signaling'
```

Comment:
This is fishy. If we're changing the order in some way, shouldn't we just do a lexicographical sort?